common/src/main/java/com/wynntils/core/features/overlays/annotations/OverlayInfo.java

```diff
@@ -13,7 +13,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 public @interface OverlayInfo {
-    /* Whether the annotated overlay instantiated at registration */
+    /* Whether the annotated overlay enabled */
```

Comment:
```suggestion
    /* Whether the annotated overlay is enabled by default */
```

Reply:
Is that actually what OverlayInfo#enabled is? We will use that field to save Overlay enabled config status too (as OverlayManager is in-memory only).  So we will have OverlayManager and OverlayInfo both have the state of the Overlay? I guess we should deal with this in the overlay configs PR.

Reply:
I feel like removing enabled/disabled state from the manager would be the easiest, we don't even need reflection as OverlayInfo is cached to a map already.

Reply:
Anyway, problem of the future, but I wouldn't change that comment.

Reply:
Well, now I'm not sure what this means either. :-) What have you done. :) Ok, fine, leave the comment be. We'll make this better when fixing overlay config.